index.bs

```diff
@@ -8345,61 +8418,39 @@ across those interfaces.
 Note: This is because all of the members of the interface and its consequential
 interfaces get flattened down on to the object that implements the interface.
 
-The [{{Global}}] and
-[{{PrimaryGlobal}}] extended attributes
+The [{{Global}}] extended attribute
 can also be used to give a name to one or more global interfaces,
 which can then be referenced by the [{{Exposed}}]
 extended attribute.
 
-The [{{Global}}] and
-[{{PrimaryGlobal}}]
-extended attributes must either
-[=takes no arguments|take no arguments=]
-or [=takes an identifier list|take an identifier list=].
+The [{{Global}}] extended attribute must either
+[=takes no arguments|take no arguments=],
```

Comment:
I don't really have an opinion about this. This was how it was specified before (maybe implicitly, just for PrimaryGlobal).